scripts/runtime/Freeze-RuntimeInputPacket.ps1

```
@@ -194,21 +256,36 @@ function Get-VibeSpecialistBindingProfile {
+            planning = 40
+            deliverable = 30
+            verification = 20
+            implementation = 10
+        }
+        $bestMatchScore = 0
+        $bestPriority = -1
         foreach ($profileName in @('planning', 'implementation', 'deliverable', 'verification')) {
             if (-not ($profiles.PSObject.Properties.Name -contains $profileName)) {
                 continue
             }
             $profile = $profiles.$profileName
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'profile' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)